README.md

```diff
@@ -78,13 +78,13 @@ This will set up a new application on your Heroku account using this repo as a t
 
 -->
 
-### Import to Replit
+### Import to CodeSandbox
 
-Create a Replit account at [Replit - Sign up](https://replit.com/signup), and then click on this button:
+Log in or register on [CodeSandbox - Sign in](https://codesandbox.io/signin?landingpage).
```

Comment:
what is `?landingpage`

Reply:
Updated the link. I copied it from the URL and never questioned what this is. I updated it and removed `?landingpage`

[5df4c00](https://github.com/upleveled/express-guest-list-api-memory-data-store/pull/294/commits/5df4c00f5fac34caac13f4001bceb87bcc5a6903)